src/main/java/com/exadel/discount/entity/Category.java

```diff
@@ -0,0 +1,40 @@
+package com.exadel.discount.entity;
+
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.*;
+import java.util.List;
+import java.util.UUID;
+
+@Getter @Setter
+@EqualsAndHashCode
+@NoArgsConstructor
+@Entity
+@Table(name = "categories")
+public class Category {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private UUID id;
```

Comment:
You can use this
@GeneratedValue(generator = "UUID")
@GenericGenerator(name = "UUID", strategy = "org.hibernate.id.UUIDGenerator")
for UUID

Reply:
+